Added reference to MDE (#9068)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/report-junk-email-messages-to-microsoft.md

ms.assetid: c31406ea-2979-4fac-9288-f835269b9d2f
ms.collection: 
  - M365-security-compliance

> description: Admins can learn about the different ways to report good and bad messages, urls, email attachments and admins to Microsoft for analysis.

Reviewer: ```suggestion
description: Admins can learn about the different ways to report good and bad messages, URLs, email attachments, and admins to Microsoft for analysis.
```

- [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
- [Microsoft 365 Defender](../defender/microsoft-365-defender.md)

> In Microsoft 365 organizations with mailboxes in Exchange Online or standalone Exchange Online Protection (EOP) organizations without Exchange Online mailboxes, both users and admins have several different methods for reporting email messages, URLs and email attachments to Microsoft. 

Reviewer: ```suggestion
In Microsoft 365 organizations with mailboxes in Exchange Online or standalone Exchange Online Protection (EOP) organizations without Exchange Online mailboxes, users and admins have several different methods for reporting email messages, URLs, and email attachments to Microsoft. 
```

> Even Microsoft 365 organizations with Microsoft Defender for Endpoint admins have several methods for reporting files.

Reviewer: ```suggestion
In addition, Microsoft 365 organizations with Microsoft Defender for Endpoint admins also have several methods for reporting files.
```
